translationCoreApps/ScripturePane
# Refactor Fetch Data

#### This pull request addresses:
This PR allows us to run fetch datas and send progress to the core app.

#### How to test this pull request:
Try and load a tool. Right now the UI is getting blocked so you cannot check the progress but if you console app js you can confirm it is working.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/translationcoreapps/scripturepane/29)
